Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataParametersFactory.cs

```diff
@@ -115,13 +114,52 @@ public IDictionary<string, Parameter> Create(bool isCompositeContext)
                     {
                         name = "totalCount",
                         description =
-                            "Indicates if the total number of items available should be returned in the 'Total-Count' header of the response.  If set to false, 'Total-Count' header will not be provided.",
+                            "Indicates if the total number of items available should be returned in the 'Total-Count' header of the response.  If set to false, 'Total-Count' header will not be provided. Must be false when using key set paging.",
```

Comment:
This text appears here: 

![image](https://github.com/user-attachments/assets/0dd3befc-9ac5-43d7-b471-44cc6cb9cfd4)

Reply:
This change looks good

Reply:
We will be referring to this paging approach as "cursor paging" -- so this message should read something like this:
`Must be false when using cursor paging (with pageToken).`

Reply:
Changed.